src/styles/index.scss

```diff
@@ -1,3 +1,267 @@
-body {
+:root {
+  --bg-color: #00acdc;
+  --count-size-monitor: 1;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.header {
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding-right: 50px;
+  display: flex;
+  box-shadow: 0 2px 4px #0000000d;
+}
+
+.header__logo {
+  height: 60px;
+}
+
+.header__logo-moyo {
+  height: 40px;
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.nav {
+  justify-content: end;
+  align-items: center;
+  height: 60px;
+  display: flex;
+}
+
+.nav__list {
+  flex-wrap: nowrap;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item {
+  margin-right: 20px;
+  padding: 0;
+  position: relative;
+  line-height: 14.06px;
+}
+
+.nav__item-button {
+  text-align: center;
+  color: #060b35;
+  cursor: pointer;
+  height: 100%;
+  line-height: 60px;
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__item-button:hover,
+.nav__item .is-active {
+  color: var(--bg-color);
+}
+
+.is-active::after {
+  content: '';
+  background-color: var(--bg-color);
+  opacity: 1;
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.main {
+  justify-content: center;
+  height: 100vh;
+  display: flex;
+}
+
+.main__container {
+  grid-template-columns: repeat(var(--count-size-monitor), 200px);
+  grid-template-rows: repeat(406px);
```

Comment:
The `grid-template-rows` property is incorrectly defined. The `repeat` function is used incorrectly here. It should specify the number of rows and their size, e.g., `repeat(2, 200px)`. Please correct this to ensure proper grid layout.